street_network/src/render.rs

```diff
@@ -25,23 +25,18 @@ impl StreetNetwork {
 
         let mut pairs = Vec::new();
 
-        // Add a line-string and polygon per road
+        // Add a polygon per road
         for (id, road) in &initial_map.roads {
-            let properties = make_props(&[
```

Comment:
I took out the road center line. This doesn't have much use in rendering probably. As we refine the JS/anything API to osm2streets, we can expose this if it's useful